src/main/java/uk/gov/justice/digital/delius/transformers/OrderManagerTransformer.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.delius.transformers;
+
+import uk.gov.justice.digital.delius.data.api.OrderManager;
+
+import java.util.Optional;
+
+public class OrderManagerTransformer {
+
+    public static OrderManager orderManagerOf(uk.gov.justice.digital.delius.jpa.standard.entity.OrderManager entity){
+        return OrderManager.builder().
+            name(Optional.ofNullable(entity.getStaff().getForename()).orElse("").
+                concat(" "+Optional.ofNullable(entity.getStaff().getSurname()).orElse(""))).
+            staffCode(Optional.ofNullable(entity.getStaff().getStaffId()).map(staffId->staffId.toString()).orElse("")).
+            dateStartOfAllocation(entity.getTeam().getStartDate()).
+            dateEndOfAllocation(entity.getTeam().getEndDate()).
+            officerId(entity.getOrderManagerId()).
+            probationAreaId(entity.getProbationArea().getProbationAreaId()).
+            teamId(entity.getTeam().getTeamId()).build();
```

Comment:
Minor thing - any reason for putting the `.` at the end of the line here?  Other places in the project seems to put it at the start.

Reply:
ok not an issue , I think is just agreed style. I ll put  them on the other side.